### ISSUE TITLE
Check for inconsistencies between uml class diagrams and tables

### ISSUE THREAD

**Issue by maxcollombin** [opengeospatial/styles-and-symbology#78]
e.g., for the [Metadata](https://github.com/opengeospatial/styles-and-symbology/blob/main/1-core/sections/clause_7_core.adoc?plain=1#L7-L54) class

![image](https://github.com/user-attachments/assets/ac73c7a9-c610-407f-84e4-f47d0a0ea395)

**Comment by maxcollombin** [comment]
`Capability System Identifiers` should be connected to `SystemIdentifierExpression`.

![Capability System Identifiers](https://github.com/user-attachments/assets/84d21b4d-91ab-4da9-9915-32287198e72b)

**Comment by jerstlouis** [comment]
Yes we should have a `capabilities` in the `SystemIdentifierExpression` enumeration I think?

The `capabilities: Capabilities` It's just like `Visualization` and `DataLayer`.

But to split it in a separate diagram, we could just add `capabilities` to the enum, and an empty `Capabilities` box, and the expanded `Capabilities` box below as we have now.

**Comment by maxcollombin** [comment]
Change [metadata name](https://github.com/opengeospatial/styles-and-symbology/blob/main/1-core/sections/clause_7_core.adoc?plain=1#L48) to `identifier` and update the Definition as follow: `An identifier to reference the Style`

**Comment by ebocher** [comment]
Done ?
